Orthtree/doc/Orthtree/Orthtree.txt

```diff
@@ -55,29 +57,27 @@ the existing ones do not match users' needs.
 
 \subsection Section_Orthtree_Quadtree Building a Quadtree
 
-The `Orthtree` class may be templated with `Orthtree_traits_2` and thus
-behave as a %quadtree. For convenience, the alias `Quadtree` is provided.
+The `Orthtree` class may be templated with `Orthtree_traits_point<>`
+while specifying a 2d ambient space and thus behave as a %quadtree.
```

Comment:
why did you remove the s at behave (actually you remove it line 75 but the question is valid here too)?

Reply:
This is from Jackson and I think behave is the infinitive here and not the 3rd person.